fiftyone/server/routes/frames.py

```
@@ -20,6 +21,18 @@
 import fiftyone.server.view as fosv
 
 
+def _heavy_frame_paths(view):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_heavy_frame_paths`

(ANN202)